docs/DeveloperGuide.md

```diff
@@ -289,82 +289,98 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `Cher` and the **Actor** is the `User`, unless specified otherwise)
 
-**Use case: Add a contact**
+#### Use case: UC1 - Add contact
 
 **MSS**
 
-1. User requests to add a contact using the command format: add n/[name] a/[address] c/[contact]
-2. Addressbook checks if command is in the valid format 
-3. Addressbook adds the contact to the existing list
-4. Addressbook displays a success message to the user: [Contact name] successfully added to list.
+1. User chooses to add contact.
```

Comment:
Is this selection step necessary (UC1/2/3 step 1)? I'm wondering if it would be easier to just start from step 2, since we do not have this selection interface for the user

Reply:
Yea, I wondered the same thing. I'm saw the guide and I don't know if they put it there because of UI selection or they want to show intent.